doc/pattern_tools/gen_pattern.py

```diff
@@ -33,22 +33,29 @@ def __init__(self, cols, rows, output, units, square_size, radius_rate, page_wid
         self.height = page_height
         self.g = SVG("g")  # the svg group container
 
-    def make_circles_pattern(self):
+    def make_acircles_pattern(self):
```

Comment:
Repair names.

Reply:
Fixed copy-paste error from experimental. Function names and function order should match the original file.